esrally/metrics.py

```diff
@@ -1741,6 +1748,20 @@ def ml_processing_time_stats(self):
                 })
         return result
 
+    def total_transform_metric(self, metric_name):
+        values = self.store.get_raw(metric_name)
+        result = []
+        if values:
+            for v in values:
+                transform_id = v.get("meta", {}).get("transform_id")
+                if transform_id is not None:
+                    result.append({
+                        "id": transform_id,
+                        "value": v["value"],
```

Comment:
should this be `"mean": v["value"]` instead, to conform with other parts?

Reply:
I don't have a strong preference but I agree that `mean` conveys clearly how the summary metric has been calculated.